### PR TITLE
Adding RFC 5888 update text

### PR DESCRIPTION
**Pull request by cdh4u**

### CODE REVIEW

File: draft-ietf-mmusic-sdp-bundle-negotiation.xml [hunk 1]
```diff
@@ -1845,7 +1849,30 @@ SDP Answer
     </section>
   </section>
 
+  <section anchor="sec-5888" title="Update to RFC 5888" toc="default">
+      <t>
+        This section updates RFC 5888 <xref format="default" pageno="false" target="RFC3261"/>),
```
Comment (rshpount) [inline draft-ietf-mmusic-sdp-bundle-negotiation.xml]: Should be a reference to RFC 5888, not 3261,

File: draft-ietf-mmusic-sdp-bundle-negotiation.xml [hunk 2]
```diff
@@ -1845,7 +1849,30 @@ SDP Answer
...
+  <section anchor="sec-5888" title="Update to RFC 5888" toc="default">
+      <t>
+        This section updates RFC 5888 <xref format="default" pageno="false" target="RFC3261"/>), 
+        in order to allow extensions to allow an SDP 'group' attribute containing
+        an identification-tag that identifies a "m=" section with the port set to zero
 
+        Section 9.2 (Group Value in Answers) of RFC 5888 is updated.
+    <section anchor="sec-5888-old-9_2" title="Original text of section 9.2 (3rd paragraph) of RFC 5888" toc="default">
+      <t>
+        SIP entities refuse media streams by setting the port to zero in the corresponding
```
Comment (rshpount) [inline draft-ietf-mmusic-sdp-bundle-negotiation.xml]: New and old text are swapped

File: draft-ietf-mmusic-sdp-bundle-negotiation.xml (hunk 1 above)
Comment (cdh4u) [reply draft-ietf-mmusic-sdp-bundle-negotiation.xml]: Will fix.

File: draft-ietf-mmusic-sdp-bundle-negotiation.xml (hunk 2 above)
Comment (cdh4u) [reply draft-ietf-mmusic-sdp-bundle-negotiation.xml]: Will fix.

File: draft-ietf-mmusic-sdp-bundle-negotiation.xml (hunk 1 above)
Comment (rshpount) [reply draft-ietf-mmusic-sdp-bundle-negotiation.xml]: Looks good

File: draft-ietf-mmusic-sdp-bundle-negotiation.xml (hunk 2 above)
Comment (rshpount) [reply draft-ietf-mmusic-sdp-bundle-negotiation.xml]: Looks good as well

**Pull request merged**